shenyu-bootstrap/src/main/resources/application.yml

```diff
@@ -86,6 +86,39 @@ shenyu:
         writeSpinCount: 16
         autoRead: true
         allocType: "pooled"
+  #  httpclient:
+  #    strategy: webClient
+  #    connectTimeout: 45000
+  #    responseTimeout: 3000
+  #    readerIdleTime: 3000
+  #    writerIdleTime: 3000
+  #    allIdleTime: 3000
+  #    readTimeout: 3000
+  #    writeTimeout: 3000
+  #    wiretap: false
+  #    keepAlive: false
+  #    pool:
+  #      type: ELASTIC
+  #      name: proxy
+  #      maxConnections: 16
+  #      acquireTimeout: 45000
+  #    proxy:
+  #      host:
+  #      port:
+  #      username:
+  #      password:
+  #      nonProxyHostsPattern:
+  #    ssl:
+  #      useInsecureTrustManager: true
+  #      keyStoreType: PKCS12
+  #      keyStorePath: classpath:keystore.p12
+  #      keyStorePassword: 123456
+  #      keyPassword: 123456
+  #      trustedX509Certificates:
+  #      handshakeTimeout:
+  #      closeNotifyFlushTimeout:
+  #      closeNotifyReadTimeout:
+  #      defaultConfigurationType:      
```

Comment:
maybe need keyStoreProvider

/** 
         * Keystore provider for netty httpClient and webclient.
         */
        private String keyStoreProvider;